mgmtsystem_action/views/mgmtsystem_action.xml

```diff
@@ -0,0 +1,194 @@
+                    </group>
+                    <newline/>
+                    <group expand="0" string="Group By...">
+                        <filter string="User" icon="terp-personal" domain="[]" context="{'group_by':'user_id'}"/>
+                        <separator orientation="vertical"/>
+                        <filter string="Type" icon="terp-personal" domain="[]" context="{'group_by':'type_action'}"/>
+                    </group>
+                </search>
+            </field>
+        </record>
+
+        <record id="view_mgmtsystem_action_form" model="ir.ui.view">
+            <field name="name">mgmtsystem.action.form</field>
+            <field name="model">mgmtsystem.action</field>
+            <field name="arch" type="xml">
+                <form string="Action">
+                    <header>
+                        <field name="stage_id" widget="statusbar" clickable="True"/>
+                    </header>
+                    <sheet string="Action">
+                        <group colspan="4" col="4">
+                            <field name="name" colspan="4" string="Subject"/>
+                            <field name="type_action" required="True"/>
+                            <field name="reference"/>
+                            <field name="user_id" string="Responsible" required="1"/>
+                            <field name="date" string="Date"/>
+                            <field name="date_deadline" string="Deadline"/>
+                            <field name="system_id"/>
+                            <field name="company_id" groups="base.group_multi_company"/>
+                        </group>
+                        <notebook colspan="4">
+                            <page string="Description">
+                                <field name="description" colspan="4" nolabel="1"/>
+                            </page>
+                        </notebook>
+                    </sheet>
+                    <div class="oe_chatter">
+                        <field name="message_follower_ids" widget="mail_followers" groups="base.group_user"/>
+                        <field name="message_ids" widget="mail_thread"/>
+                    </div>
+                </form>
+            </field>
+        </record>
+
+        <record id="view_mgmtsystem_action_kanban" model="ir.ui.view">
+            <field name="name">mgmtsystem.action.kanban</field>
+            <field name="model">mgmtsystem.action</field>
+            <field name="arch" type="xml">
+                <kanban default_group_by="stage_id" class="o_kanban_small_column">
+                    <field name="name"/>
+                    <field name="type_action"/>
+                    <field name="reference"/>
+                    <field name="user_id"/>
+                    <field name="date"/>
+                    <field name="date_deadline"/>
+                    <field name="message_needaction_counter"/>
```

Comment:
AFAICR to be able to use this, you need to also inherit from the need action mixin model.